DOCUMENTATION.md

```diff
@@ -485,9 +485,10 @@ This is an invitation-only feature. Contact the GOV.UK Notify team on the [suppo
 ### Method
 
 ```php
-response = notifications_client.send_precompiled_letter_notification(
+$response = $notifyClient->sendPrecompiledLetter(
     $reference,
-    $pdf_file
+    $file_contents,
```

Comment:
I'd rename this to `$pdf_data` here and in the arguments list below to be consistent with the function argument name in `Client.php`